[DOCS] Add SECURITY.md and update README.md. (#906)
authelia/authelia: SECURITY.md

> Security is taken very seriously here, therefore we follow the rule of responsible
> disclosure and we encourage you to do so.

Reviewer: ```suggestion
Authelia takes security very seriously. We follow the rule of [responsible disclosure](https://en.wikipedia.org/wiki/Responsible_disclosure), and we encourage the community to as well.
```

Thoughts?

Author: yeah, I did not properly rebase I guess.

Author: I aligned all documents in the next commit. Thanks.
